doc/source/security.md

```diff
@@ -155,3 +155,44 @@ This is a sensitive security issue (similar to writing sudo rules in a
 traditional computing environment), so be very careful.
 
 There's ongoing work on making this easier!
+
+
+## Kubernetes Network Policies
+
+Kubernetes has optional support for [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) which let you restrict how pods can communication with each other and the outside world. This can provide additional security within JupyterHub, and can also be used to limit network access for users of JupyterHub.
+
```

Comment:
By default, the JupyterHub helm chart **disables** network policies.

### Enabling network policies

**Important:** If you decide to enable network policies, you should be aware that a Kubernetes cluster may have partial, full, or no support for network policies.  Kubernetes will **silent ignore** policies that aren't supported. If you rely on a policy to restrict what users can access you should verify the policies are enforced on your cluster. Please use **caution** if enabling network policies to make sure that the policies behave as expected.

Reply:
@willingc I've reworded your suggested text slightly to avoid repetition. Does this look OK?